lib/users/get.js

```diff
@@ -0,0 +1,19 @@
+const async = require("async");
+const _ = require("lodash");
+const crypto = require("crypto");
+
+module.exports = function(req, res) {
+    req.pgdb.query("SELECT * FROM users", (error, result) => {
+        if(error) {
+            req.log.error({ error }, "Error while getting users");
+            return res.sendStatus(500);
+        }
+        let users = result.rows.map(row => {
+            return {
+                userId: row.data.remoteId,
+                name: row.data.name
+            };
+        });
+        res.send({ users });
```

Comment:
If multiple tokens are generated for the same users, there will be duplicates in the list. You could do some kind of uniqueness-test.